app/code/Magento/ConfigurableProduct/view/adminhtml/web/js/components/dynamic-rows-configurable.js

```diff
@@ -405,7 +405,7 @@ define([
                 'id': row.productId,
                 'product_link': row.productUrl,
                 'name': $('<i></i>').text(row.name).html(),
-                'sku': $('<i></i>').text(row.sku).html(),
```

Comment:
```suggestion
                row.sku = $('<i></i>').html(row.sku).html();
```

Reply:
Hello @swnsma,

If the SKU is like "C&A" with your solutions "$('<i></i>').html('C&A').html();" is become "Best &amp; Test" and that not the correct solution for it.

Thanks

Reply:
In this point - yes.
I am not happy with `$('<i></i>')` anyway, so it is better option to check if issue which was fixed by adding `$('<i></i>')` will not be reproduced in this PR.